homeassistant/components/switch/rainmachine.py

```diff
@@ -176,16 +246,16 @@ def _update(self) -> None:
         except exceptions.HTTPError as exc_info:
             _LOGGER.error('Unable to update info for program "%s"',
                           self.rainmachine_id)
-            _LOGGER.debug(str(exc_info))
+            _LOGGER.debug(exc_info)
 
 
 class RainMachineZone(RainMachineEntity):
     """A RainMachine zone."""
 
-    def __init__(self, client, zone_json, **kwargs):
+    def __init__(self, client, zone_json, zone_run_time, **kwargs):
         """Initialize a RainMachine zone."""
-        super(RainMachineZone, self).__init__(client, zone_json, **kwargs)
-        self._run_time = kwargs.get(CONF_ZONE_RUN_TIME)
+        super().__init__(client, zone_json, **kwargs)
+        self._run_time = kwargs.get(zone_run_time)
```

Comment:
That looks like what I have. But I think this line should be changed to this:
```py
self._run_time = zone_run_time
```
Cause `zone_run_time` is now available as a positional argument.